comeet/Data/Extensions/Date+API.swift

```diff
@@ -12,24 +12,28 @@ extension Date {
     
     static func fromAPI(stringDate: String) -> Date? {
         let formatter = DateFormatter()
+        formatter.timeZone = TimeZone(abbreviation: "UTC")
         formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss-SSSS"
```

Comment:
New in iOS 10: [ISO8601DateFormatter](https://developer.apple.com/reference/foundation/iso8601dateformatter/)